lametro/views.py

```diff
@@ -80,9 +80,11 @@ def get_context_data(self, **kwargs):
         except:
             context['packet_url'] = None
 
-        # Create list of related board reports
+        # Create list of related board reports, ordered by descending last_action_date.
         if context['legislation'].related_bills.all():
-            context['related_bills'] = [LAMetroBill.objects.get(identifier=bill.related_bill_identifier) for bill in context['legislation'].related_bills.all()]
+            related_bills = [LAMetroBill.objects.get(identifier=bill.related_bill_identifier) for bill in context['legislation'].related_bills.all()]
```

Comment:
while we're in here, let's make this a bit more efficient. this approach will make a database call for each related bill. how about we make use of [the `in` filter](https://docs.djangoproject.com/en/2.1/ref/models/querysets/#in) to grab all the `LAMetroBill` objects where the identifier is in the related bills?